### PR TITLE
Test views

### PR DESCRIPTION
**Pull request by bowlerct**
Added tests for django views

### CODE REVIEW

**Review by SchoolGuy — APPROVED** [review]
I will hit the LGTM button for this so hard that my mouse probably breaks. This is huge! Thanks!

File: tests/views/actions_view_test.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import pytest
+from django.urls import reverse
+
+# Test action requests in view. These test the website operation
```
Comment (dansou901) [inline tests/views/actions_view_test.py:4]: "These tests"...

Comment (dansou901) [inline tests/views/actions_view_test.py:4]: "Tests action..."

File: tests/views/list_items_view_test.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import pytest
+from django.urls import reverse
+from cobbler import api as cobbler_api
+
+# Test listing of items
```
Comment (dansou901) [inline tests/views/list_items_view_test.py:5]: "Tests listing of items"

**Review by dansou901 — CHANGES_REQUESTED** [review]
See inline comment

File: tests/views/list_items_view_test.py (hunk 2 above)
Comment (bowlerct) [reply tests/views/list_items_view_test.py]: updated

**Review by dansou901 — APPROVED** [review]
LGTM now.

**Review by hbokh — APPROVED** [review]
If it works, it works!! LGTM

**Pull request merged**